stake-pool/program/src/stake.rs

```diff
@@ -102,6 +103,55 @@ pub enum StakeInstruction {
     AuthorizeWithSeedNOTUSED,
 }
 
+/// FIXME copied from the stake program
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Copy)]
+#[allow(clippy::large_enum_variant)]
+pub enum StakeState {
+    /// FIXME copied from the stake program
```

Comment:
Just curious: why the FIXME comments on some variants and fields as well as the enum/struct?

Reply:
Clippy requires public enums / structs / fields / functions to have an annotation, so we just throw this one on usually

Reply:
Ah thanks, forgot about that setting on this repo